databricks_cli/sdk/api_client.py

```diff
@@ -127,6 +128,11 @@ def perform_query(self, method, path, data = {}, headers = None):
             except ValueError:
                 pass
             raise requests.exceptions.HTTPError(message, response=e.response)
+
+        # delete returns nothing.
+        if method.lower() == 'delete':
```

Comment:
not everywhere. please remove.

Reply:
I'll have to play with this.
I think delete returns an empty body on the SCIM api, and that's the issue.